src/Builder.php

```diff
@@ -95,20 +98,24 @@ public function __construct(
      */
     public function canOnlyBeUsedBy($audience, $replicateAsHeader = false)
     {
-        return $this->permittedFor($audience, $replicateAsHeader);
+        return $this->permittedFor($audience);
     }
 
     /**
      * Configures the audience
      *
-     * @param string $audience
-     * @param bool $replicateAsHeader
+     * @param list<string> $audiences
      *
      * @return Builder
      */
-    public function permittedFor($audience, $replicateAsHeader = false)
+    public function permittedFor(...$audiences)
```

Comment:
I think we should check whether the last `$audiences` element is a `boolean`, and inform the use that, for this method, BC broke to favor FC

Reply:
Yeap, just had the same thought before going to bed. I'll update it 👍

Reply:
@Slamdunk just applied that with a filter for string audiences only - the filter might cause minor issues but it's more compatible with new the behaviour (we can change to a string cast btw).